types/cli-welcome/index.d.ts

```diff
@@ -0,0 +1,19 @@
+// Type definitions for cli-welcome 2.2
+// Project: https://github.com/ahmadawais/cli-welcome
+// Definitions by: Max Programming <https://github.com/max-programming>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+interface WelcomeOptions {
+    title: string;
+    tagLine: string;
+    description?: string;
+    version?: string;
+    bgColor?: string;
+    color?: string;
+    clear?: boolean;
+    bold?: boolean;
+}
+
+declare function welcome(options: WelcomeOptions): void;
```

Comment:
```ts
declare function welcome(options?: WelcomeOptions): void;
```
`options `is an optional param and all props are optional, taking defaults:
https://github.com/ahmadawais/cli-welcome/blob/master/index.js#L13
I'm givin OK as  @ahmadawais revied this alrady, thx!